### ISSUE TITLE
Ruler separators are almost not visible

### ISSUE THREAD

**Issue by Supportic** [robb0wen/synthwave-vscode#61]
![Code_Q0wXfMvnjl_LI](https://user-images.githubusercontent.com/20886239/57191282-fa933280-6f23-11e9-841c-48add0d20f2d.jpg)

**Comment by tlgreg** [comment]
Yeah, currently it's `#34294f33`, but it has low contrast even without any transparency:
![without transparency](https://i.imgur.com/lMAfE8t.png)

**Comment by robb0wen** [comment]
I'll look to update this in the next release

**Comment by robhaswell** [comment]
Is there a workaround for this? Unfortunately just had to switch back off to default.

**Comment by tlgreg** [comment]
I don't currently use synthwave, but I used this custom css when I did:
```css
.monaco-editor .view-ruler {
  box-shadow: 1px 0 0 0 #34294f inset;
}
```

**Comment by Supportic** [comment]
Hmm still not fixed :/

**Comment by jasonrundell** [comment]
> ```css
> .monaco-editor .view-ruler {
>   box-shadow: 1px 0 0 0 #34294f inset;
> }
> ```

How did you add custom css on top of the synthwave theme @tlgreg ?

**Comment by jasonrundell** [comment]
> > ```css
> > .monaco-editor .view-ruler {
> >   box-shadow: 1px 0 0 0 #34294f inset;
> > }
> > ```
> 
> How did you add custom css on top of the synthwave theme @tlgreg ?

Found a workaround: 

Editor your VS Code `settings.json` and add the following lines:

```json
    "workbench.colorCustomizations": {
        "editorRuler.foreground": "#a37ee7"
    }
```

**Comment by jasonrundell** [comment]
Putting a hand up ✋ to try and do a PR for a fix to the theme json to improve the ruler visibility.

**Comment by texastoland** [comment]
In Insiders you can work around with `"editor.rulers": [{"column": 80, "color": "#34294f"}]`.
